.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -8,3 +8,19 @@ labels: kind/enhancement
 **What would you like to be added**:
 
 **Why is this needed**:
+
+<!-- Please select an area, component, kind, platform and priority for this issue. This helps the community categorizing it. -->
+<!-- Replace below TODOs or exchange the existing identifiers with those that fit best in your opinion. -->
+<!-- If multiple identifiers make sense you can also state the commands multiple times, e.g. -->
+<!--   /area control-plane -->
```

Comment:
Does the bot understands also input such as `/area control-plane` or only `/area/control-plane`?

Reply:
Both